VENTouchLock/VENTouchLock.m

```diff
@@ -174,71 +180,57 @@ - (void)requestTouchIDWithCompletion:(void (^)(VENTouchLockTouchIDResponse))comp
 
 - (void)lock
 {
-    if (![self isPasscodeSet]) {
+    if (![self isPasscodeSet] || self.lockWindow.isKeyWindow) {
         return;
     }
 
     if (self.splashViewControllerClass != NULL) {
         VENTouchLockSplashViewController *splashViewController = [[self.splashViewControllerClass alloc] init];
         if ([splashViewController isKindOfClass:[VENTouchLockSplashViewController class]]) {
-            UIWindow *mainWindow = [[UIApplication sharedApplication].windows firstObject];
-            UIViewController *rootViewController = [UIViewController ventouchlock_topMostController];
-            UIViewController *displayController;
+            self.lockWindow = [[UIWindow alloc] initWithFrame:self.mainWindow.frame];
```

Comment:
@hyperspacemark I agree with your points, my only concern on having this `obscured behavior` that (potentially) breaks existing `VENTouchLock` integrations is adding integration complexity. We should solve the issue within the framework, and not pass on the responsibility to the developers. @ayanonagon, @eliperkins what do you think?